SAKPaaS/src/app/components/user-card/user-card.component.scss

```diff
@@ -0,0 +1,25 @@
+#user-card {
+  @media screen and (max-width: 600px) {
+    display: none;
+  }
+  position: absolute;
+  top: 45px;
+  right: 30px;
+  border-radius: 10px;
+  a {
+    color: var(--greenblue);
+    text-decoration: underline;
+    cursor: pointer;
+  }
+  mat-icon {
+    display: none;
```

Comment:
I would prefer to have temporary implementations like this tagged. with `// TODO:` or something like that. But that's personal preference.